app/src/main/java/bubbleheads/buy_cook/PagerAdapter.java

```diff
@@ -4,13 +4,10 @@
 import android.support.v4.app.FragmentManager;
 import android.support.v4.app.FragmentStatePagerAdapter;
 
-/**
- * Created by melod on 26.03.2017.
- */
-
 public class PagerAdapter extends FragmentStatePagerAdapter {
 
-    int mNumOfTabs;
+    private int mNumOfTabs;
```

Comment:
why do you use m prefix for this field? it is the only place you follow this convention. Use just tabsCount or so